### ISSUE TITLE
Abnormal GPU usage of Vita3K android application itself (NOT THE GAMES) when you are on its homescreen and menus, that causes device freeze, white screen at the same time

### ISSUE THREAD

**Issue by cyemac1995** [Vita3K/Vita3K-Android#437]
![image](https://github.com/Vita3K/Vita3K-Android/assets/86140950/7d2d34bf-93f2-4b15-9071-881276caff07)
With default/custom adreno driver used (Abnormal GPU usage, always stays at 99%)

![image](https://github.com/Vita3K/Vita3K-Android/assets/86140950/2ed4411c-98f7-4419-b5db-a1ba902e5479)
With opengl and turnip custom driver used (Normal GPU usage)

Please have a look on this, I found out the emulator's abnormal usage of GPU (constant 99%, does not flactuate) when dafault/custom adreno vulkan driver is used, but when I use openGL and Turnips, the GPU usage is normal flactuates between 25% to 92%, but most of the time it stays on between 25%-50% for OpenGL and 50%-80% for Turnip driver

This heavy and full GPU usage will cause device freeze, system UI forceclose and white screen.

In the screenshot, I just launched vita3k, no games are opened

When I launch the game like One Piece using default vulkan, the GPU usage is normal, sadly, when I return to emulator's GUI after closing the game, my device freezes then white screen.

My device is redmi note 12 4g SD685 with A610 GPU.

This is not happening in release v5 and below.

I couldn't find the adb logs that reports the problems regarding GPU usage, it only shows "slow operation".

![image](https://github.com/Vita3K/Vita3K-Android/assets/86140950/e3223350-3fe7-4201-aa77-0f006937eab0)
![image](https://github.com/Vita3K/Vita3K-Android/assets/86140950/c3ccf286-3f66-4922-a244-72f56f246eb4)
![image](https://github.com/Vita3K/Vita3K-Android/assets/86140950/9a06375d-7c92-468c-a08e-cb749f1b63b6)

This is a proof that GPU usage is normal in the game using default Vulkan driver with native buffer.
It seems that the android emulator itself is like a heavy game now that uses a lot of/full GPU power constantly at 99%  with default vulkan/custom adreno

So we cannot blame the device 'cause of its low end SoC, the games run smoothly like One Piece BB, it is just the emulator's bug in handling GPU with default/custom adreno vulkan driver when you are on its GUI and menus

**Comment by HyemiChan** [comment]
Hi I have the same phone, i cant even get into the gpu settings, it just freeze, white screen then my phone suddenly restarts...

**Comment by cyemac1995** [comment]
> Hi I have the same phone, i cant even get into the gpu settings, it just freeze, white screen then my phone suddenly restarts...

yes, I already mentioned Mcdu in discord and reported this issue, I am waiting for his reply. If u want play games you need to download turnip 23.3 drivers and import it. Import it and allow your device to freeze then restart, open the app set turnip as gpu then save and close immediately, you need to do it faster before freezing starts, set screen refresh rate as 120hz first

**Comment by surlup** [comment]
> > Hi I have the same phone, i cant even get into the gpu settings, it just freeze, white screen then my phone suddenly restarts...
> 
> yes, I already mentioned Mcdu in discord and reported this issue, I am waiting for his reply. If u want play games you need to download turnip 23.3 drivers and import it. Import it and allow your device to freeze then restart, open the app set turnip as gpu then save and close immediately, you need to do it faster before freezing starts, set screen refresh rate as 120hz first

I'd rather play games that don't take up a lot of GPU, like Tales of Hearts and Trails in Sky, rather than burdening my cellphone while waiting for the next update in the future

**Comment by cyemac1995** [comment]
> > > Hi I have the same phone, i cant even get into the gpu settings, it just freeze, white screen then my phone suddenly restarts...
> > 
> > 
> > yes, I already mentioned Mcdu in discord and reported this issue, I am waiting for his reply. If u want play games you need to download turnip 23.3 drivers and import it. Import it and allow your device to freeze then restart, open the app set turnip as gpu then save and close immediately, you need to do it faster before freezing starts, set screen refresh rate as 120hz first
> 
> I'd rather play games that don't take up a lot of GPU, like Tales of Hearts and Trails in Sky, rather than burdening my cellphone while waiting for the next update in the future

It is not the game issue. It is the emulator's GUI, homescreen and menus that lock GPU usage to 99% and doesn't fluctuate. That's the reason why my device freeze then returns to white screen output due to overloading of the GPU when you are in dafault/custom adreno driver (non-turnip) ONLY. Many games actually used GPU NORMALLY, and the performance depends on the SoC itself.

**Comment by HyemiChan** [comment]
This problem is still not fix with v9 :/

**Comment by Guilherm3Rodrigues** [comment]
Not fix with v11 =S

**Comment by M45uM1** [comment]
So this is the answer of why the games would exit after some time after being booted up....
